src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs

```diff
@@ -51,7 +51,7 @@ public class CleanseLogMessage
                 new Regex(@"(?<=\?|&)(X-Plex-Client-Identifier|X-Plex-Token)=(?<secret>[^&=]+?)(?= |&|$)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
 
                 // Indexer Responses
-                new Regex(@"avistaz\.[a-z]{2,3}\\\/rss\\\/download\\\/(?<secret>[^&=]+?)\\\/(?<secret>[^&=]+?)\.torrent", RegexOptions.Compiled | RegexOptions.IgnoreCase),
+                new Regex(@"(?:avistaz|exoticaz|cinemaz)\.[a-z]{2,3}\\\/rss\\\/download\\\/(?<secret>[^&=]+?)\\\/(?<secret>[^&=]+?)\.torrent", RegexOptions.Compiled | RegexOptions.IgnoreCase),
```

Comment:
There are more I thought

Reply:
That's the format known for 2/3 and assumed for the 3rd.

What else is using that format?

Reply:
PHD is also avistaz based, so assume they are?